public/Gruntfile.js

```diff
@@ -1,66 +1,66 @@
-module.exports = function(grunt) {
+module.exports = function (grunt) {
+  grunt.initConfig({
+    pkg: grunt.file.readJSON("package.json"),
 
-  require('load-grunt-tasks')(grunt);
-    grunt.initConfig({
-        pkg: grunt.file.readJSON('package.json'),
-
-        babel: {
-          options: {
-            sourceMap: true,
-            plugins: ['transform-react-jsx'],
-            presets: ['es2015', 'react'],
```

Comment:
would it be possible to also delete the `babel-preset-react` dependency from `package.json` if it's not being used here anymore? :0